engine/access/rest/request/script_test.go

```diff
@@ -31,7 +31,7 @@ func TestScript_InvalidParse(t *testing.T) {
 }
 
 func TestScript_ValidParse(t *testing.T) {
-	arg1 := []byte(`{"type": "String", "value": "hello" }`)
+	arg1 := []byte(`{"type": "Name", "value": "hello" }`)
```

Comment:
?

Reply:
[Fixed]Goland change String and Type in a bunch of unrelated places when I renamed the fields on the MsgAuthConfig